AppointmentReminders.Web/Content/bootstrap/type.less

```diff
@@ -0,0 +1,302 @@
+//
```

Comment:
We really need to remove all the libraries. Or at least include just the minified version...